examples/orthographic.js

```diff
@@ -10,34 +10,18 @@ var extent = new itowns.Extent(
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
 
-var r = viewerDiv.clientWidth / viewerDiv.clientHeight;
-var camera = new itowns.THREE.OrthographicCamera(
-    extent.west(), extent.east(),
-    extent.east() / r, extent.west() / r,
-    0, 1000);
-
 // Instanciate PlanarView
-var view = new itowns.PlanarView(
-        viewerDiv, extent, { renderer: renderer, maxSubdivisionLevel: 10, camera: camera });
-
-var onMouseWheel = function onMouseWheel(event) {
-    var change = 1 - (Math.sign(event.wheelDelta || -event.detail) * 0.1);
-
-    var halfNewWidth = (view.camera.camera3D.right - view.camera.camera3D.left) * change * 0.5;
-    var halfNewHeight = (view.camera.camera3D.top - view.camera.camera3D.bottom) * change * 0.5;
-    var cx = (view.camera.camera3D.right + view.camera.camera3D.left) * 0.5;
-    var cy = (view.camera.camera3D.top + view.camera.camera3D.bottom) * 0.5;
+var view = new itowns.PlanarView(viewerDiv, extent, { maxSubdivisionLevel: 10 });
 
-    view.camera.camera3D.left = cx - halfNewWidth;
-    view.camera.camera3D.right = cx + halfNewWidth;
-    view.camera.camera3D.top = cy + halfNewHeight;
-    view.camera.camera3D.bottom = cy - halfNewHeight;
+var controls = new itowns.PlanarControls(view, {
+    maxAltitude: 40000000,
+});
 
-    view.notifyChange(true);
-};
+// disable the rotation
+controls.handleRotation = function _() {};
```

Comment:
I'd prefer an option in `PlanarControls` constructor to disable rotation, instead of overriding the function.
(and currently, clicking the middle button will still rotate the camera)

Reply:
Also adding a zoom-speed option would be nice: the default zoom speed is slooooow :)

Reply:
I was hesitant to change the core, but I'm ok with adding an option (damn middle button, didn't see that one). I'm tweaking a speed too !